CRM/Activity/Form/Activity.php

```diff
@@ -928,7 +928,10 @@ public function postProcess($params = NULL) {
 
     // store the submitted values in an array
     if (!$params) {
-      $params = $this->controller->exportValues($this->_name);
+      $params = $this->getSubmittedValues();
+    }
+    else {
+      CRM_Core_Error::deprecatedWarning('passing params into postProcess is deprecated. Match parent function');
```

Comment:
yeah - cos I couldn't figure out why this would be true... looks like a legacy pattern

Reply:
@eileenmcnaughton probably from things like a testSubmit function or similar back when we weren't mocking the full form?

Reply:
@seamuslee001 yeah I suspect so - would be good to remove again so this slates for removal